client/app/services/vms.service.spec.js

```diff
@@ -5,11 +5,14 @@ describe('VMS Service', () => {
           {'id':'test', 'value':'test'}
       ];
       const sort = {'id':'name','sortType':'alpha'};
+      const permissions = readJSON('tests/mock/rbac/allPermissions.json');
 
       beforeEach(module('app.services','app.shared'));
       beforeEach( () => {
-          bard.inject('VmsService', 'CollectionsApi');
+          bard.inject('VmsService', 'CollectionsApi', 'RBAC');
```

Comment:
Hmmm...are we checking the test files for StandardJS formatting?  @AllenBW did we put that in to test in Travis?  The ```;``` is not allowed by StandardJS

Reply:
We aren't checking test files @chriskacerguis

Reply:
he aint lying, we weren't before, wasn't going to add it in the conversion (https://github.com/ManageIQ/manageiq-ui-service/blob/master/package.json#L199-L200)